patches/net/minecraft/world/entity/LivingEntity.java.patch

```diff
@@ -318,9 +346,20 @@
                  this.playSound(soundtype.getFallSound(), soundtype.getVolume() * 0.5F, soundtype.getPitch() * 0.75F);
              }
          }
-@@ -1675,9 +_,9 @@
+@@ -1641,7 +_,7 @@
+     protected void doHurtEquipment(DamageSource p_330843_, float p_330394_, EquipmentSlot... p_331314_) {
+         if (!(p_330394_ <= 0.0F)) {
+             int i = (int)Math.max(1.0F, p_330394_ / 4.0F);
+-
```

Comment:
Same as the prior comment: re-add the empty line so the patch is a pure addition